lib/OpenQA/Utils.pm

```diff
@@ -425,28 +393,18 @@ sub find_labels {
     return \@labels;
 }
 
-sub find_flags ($text) {
```

Comment:
Did you delete this function on purpose?

Reply:
oops. That seems to be a rebase problem. Thx, fixed